flink-table-store-spark/src/main/java/org/apache/flink/table/store/spark/SparkCatalog.java

```diff
@@ -68,6 +73,20 @@ public String name() {
         return name;
     }
 
+    @Override
+    public void createNamespace(String[] namespace, Map<String, String> metadata)
+            throws NamespaceAlreadyExistsException {
+        Preconditions.checkArgument(
+                isValidateNamespace(namespace),
+                "Namespace %s is not valid",
+                CatalogV2Implicits.NamespaceHelper(namespace).quoted());
```

Comment:
Don't rely on an internal api `CatalogV2Implicits`.
We can just use `Arrays.toString(...)`.